I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 305, in script call
    CS "Take this!"
  File "renpy/common/00console.rpy", line 765, in script call
    python in _console:
ScriptError: could not find label 'credits'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 305, in script call
    CS "Take this!"
  File "renpy/common/00console.rpy", line 765, in script call
    python in _console:
  File "<string>", line 1, in script
  File "C:\Users\Xander\Documents\renpy-6.99.13-sdk\renpy\ast.py", line 1552, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\Xander\Documents\renpy-6.99.13-sdk\renpy\script.py", line 861, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'credits'.

Windows-8-6.2.9200
Ren'Py 6.99.13.2919
CS Bounciness Vol. 2 1.0RC1
